utils/profile.ts

```diff
@@ -52,3 +77,85 @@ export const GetLimitedProfileByPOSTPayloadMiddleware: IRequestMiddleware<
       ResponseErrorFromValidationErrors(GetLimitedProfileByPOSTPayload)
     )
   );
+
+export type IGetLimitedProfileResponses =
+  | IResponseSuccessJson<LimitedProfile>
+  | IResponseErrorNotFound
+  | IResponseErrorQuery
+  | IResponseErrorForbiddenNotAuthorizedForRecipient;
+
+export const getLimitedProfileTask = (
+  apiAuthorization: IAzureApiAuthorization,
+  userAttributes: IAzureUserAttributes,
+  fiscalCode: FiscalCode,
+  profileModel: ProfileModel,
+  disableIncompleteServices: boolean,
+  incompleteServiceWhitelist: ReadonlyArray<ServiceId>
+  // eslint-disable-next-line max-params
+): Task<IGetLimitedProfileResponses> =>
+  taskEither
+    .of<
+      | IResponseErrorForbiddenNotAuthorizedForRecipient
+      | IResponseErrorNotFound
+      | IResponseErrorQuery,
+      void
+    >(void 0)
+    .chainSecond(
+      // Sandboxed accounts will receive 403
+      // if they're not authorized to send a messages to this fiscal code.
+      // This prevents leaking the information, to sandboxed account,
+      // that the fiscal code belongs to a subscribed user
+      fromEither(
+        canWriteMessage(
+          apiAuthorization.groups,
+          userAttributes.service.authorizedRecipients,
+          fiscalCode
+        )
+      ).mapLeft(_ => ResponseErrorForbiddenNotAuthorizedForRecipient)
```

Comment:
```suggestion
      ).mapLeft(ResponseErrorForbiddenNotAuthorizedForRecipient)
```

Reply:
`mapLeft` need a function and `ResponseErrorForbiddenNotAuthorizedForRecipient` is not